packages/wonder-blocks-modal/components/modal-launcher.test.js

```diff
@@ -73,10 +73,10 @@ describe("ModalLauncher", () => {
         wrapper.find("button").simulate("click");
         expect(document.querySelector("[data-modal-child]")).toBeTruthy();
 
-        // Simulate an Escape keypress. This will happen synchronously, because
-        // we're using `dispatchEvent`.
+        // Simulate an Escape keypress.
         const event: KeyboardEvent = (document.createEvent("Event"): any);
         event.key = "Escape";
+        event.which = 27;
```

Comment:
What about `keyCode`?  Also, is `key` still necessary now that we're not checking for it?

Reply:
This is just a test file, so I wasn't sure. However, note that `keyCode` is obsolete (https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode).

Reply:
Looks like we should be using `code`. I'll make changes accordingly so that we support things properly.